x-pack/plugins/security_solution/public/types.ts

```diff
@@ -43,6 +44,7 @@ export interface StartPlugins {
 
 export type StartServices = CoreStart &
   StartPlugins & {
+    ml: MlPluginStart;
```

Comment:
Had to add this here because if not Typescript will have problems with  `useKibana().services.ml`

Reply:
You shouldn't add `MlPluginStart` to other plugins

Reply:
@darnautov can you please elaborate?

i believe we should be adding `ml` to their `StartPlugins` here, which will eventually make it into `StartServices`

>You shouldn't add MlPluginStart to other plugins

If we want other plugins to be using our url generator, we'll need to give them access to it via our setup or start contract